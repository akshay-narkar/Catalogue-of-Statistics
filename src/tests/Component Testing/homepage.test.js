/*eslint-disable*/
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import React from 'react';
import Homepage from '../../Containers/homepage';
import store from './store';

it('UI is unaltered wrt first snapshot', () => {
  const elem = renderer.create(<BrowserRouter><Provider store={store}><Homepage /></Provider></BrowserRouter>).toJSON();

  expect(elem).toMatchSnapshot();
});