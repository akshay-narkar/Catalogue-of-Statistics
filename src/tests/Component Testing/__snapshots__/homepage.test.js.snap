// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI is unaltered wrt first snapshot 1`] = `
Array [
  <h1
    className="text-center my-2 py-3"
  >
    The Football Catalogue
  </h1>,
  <div
    className="container"
  >
    <div
      className="row my-4"
    >
      <div
        className="col-md"
      >
        <div
          className="ms-5  my-3"
        >
          <label
            className="form-label d-flex align-items-baseline m-0"
            htmlFor="exampleselectInput1"
          >
            <p
              className="m-0"
            >
              CATEGORIES:
            </p>
            <select
              aria-label="selectcategory"
              className="form-select ms-2"
              onChange={[Function]}
              value="All"
            >
              <option
                defaultValue={true}
                value="All"
              >
                All
              </option>
              <option
                label="Brazil"
                value="Brazil"
              >
                Brazil
              </option>
              <option
                label="England"
                value="England"
              >
                England
              </option>
              <option
                label="France"
                value="France"
              >
                France
              </option>
              <option
                label="Germany"
                value="Germany"
              >
                Germany
              </option>
              <option
                label="Italy"
                value="Italy"
              >
                Italy
              </option>
              <option
                label="Netherlands"
                value="Netherlands"
              >
                Netherlands
              </option>
              <option
                label="Portugal"
                value="Portugal"
              >
                Portugal
              </option>
              <option
                label="Spain"
                value="Spain"
              >
                Spain
              </option>
            </select>
          </label>
        </div>
      </div>
      <div
        className="col-md"
      >
        <div
          className="ms-5 my-3"
        >
          <label
            className="form-label d-flex align-items-baseline"
            htmlFor="exampleFormControlInput1"
          >
            <p
              className="m-0"
            >
              Team Name:
            </p>
            <input
              className="form-control text-box ms-2 w-50"
              id="exampleFormControlInput1"
              onChange={[Function]}
              placeholder="Title"
              type="text"
              value=""
            />
          </label>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="d-flex flex-wrap container p-0 m-auto justify-content-center border"
  >
    <h5>
      Loading... (API might take time. Only 10 Requests/Min)
    </h5>
  </div>,
]
`;
